Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fastcrypto"
-version = "0.1.3"
+version = "1.0.0"
```

Comment:
MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backwards compatible manner
PATCH version when you make backwards compatible bug fixes

^ following what semver CI is set to - since the API did change, we should probably increment MAJOR